src/main/java/com/technicjelle/UpdateChecker.java

```diff
@@ -7,26 +7,27 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 import java.net.MalformedURLException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.ExecutionException;
```

Comment:
IntelliJ is telling me this one isn't necessary

Reply:
fixed